WTWSMS/common/nicknames/VIET_immersion_nicknames.txt

```diff
@@ -254,13 +267,17 @@ nick_the_anger_filled = {
 
 nick_the_sage = {
 	allow = {
-		religion = buddhist
+		OR = {
+			religion = buddhist
+			religion = melchisedechianism
```

Comment:
How? It's simply "the Sage". Melchisedechianism (and Tzadikites for that metter) both place extreme value on the same mysticism that would earn Buddhism this name.